src/ui/index.js

```diff
@@ -10,15 +10,17 @@ import {i18n} from '../lib/native/I18n'
 Vue.mixin(i18nPlugin)
 Vue.mixin(capacitor)
 
-i18n.setLocales(navigator.languages)
-i18n.load()
-
-const app = (global['Floccus'] = new Vue({
-  el: '#app',
-  store,
-  router,
-  vuetify,
-  render: (h) => h(App),
-}))
+const app = new Promise(() => {
+  i18n.setLocales(navigator.languages)
+  await i18n.load()
```

Comment:
Does this cause a noticable delay?

Reply:
It also doesn't compile: https://github.com/floccusaddon/floccus/runs/4992476670?check_suite_focus=true#step:6:318

Is this strictly necessary for the changes above to work?

Reply:
Sorry for my mistake. Too busy not fully tested. Repairs have been reworked.

`i18n.load()` returns a Promise, you must make sure that this Promise has completed before calling `t()`, otherwise it may return the default `en` string.

Waiting for `i18n.load()` to return, there will be a delay of 3~5ms, which I think is almost negligible.

Reply:
> Waiting for i18n.load() to return, there will be a delay of 3~5ms

Alright, that's fine